tests/v2/validation/provisioning/rke1/provisioning_cloud_provider_test.go

```diff
@@ -104,7 +106,7 @@ func (r *RKE1CloudProviderTestSuite) TestAWSCloudProviderRKE1Cluster() {
 
 func (r *RKE1CloudProviderTestSuite) TestVsphereCloudProviderRKE1Cluster() {
 	nodeRolesDedicated := []provisioninginput.NodePools{provisioninginput.EtcdNodePool, provisioninginput.ControlPlaneNodePool, provisioninginput.WorkerNodePool}
-	nodeRolesDedicated[0].NodeRoles.Quantity = 3
+	nodeRolesDedicated[0].NodeRoles.Quantity = 1
```

Comment:
Why is this changed to 1?

Reply:
I was running into trouble with resource usage in vsphere 7 setup, and technically you don't need 3 etcd for this test. Good callout though I can change it back to 3.